<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>74</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Monitors">
     <item>
      <layout class="QHBoxLayout" name="Energy_monitor">
       <item>
        <widget class="QLabel" name="Energy_text">
         <property name="text">
          <string>Energy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Energy_value">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Impulse_monitor">
       <item>
        <widget class="QLabel" name="Impulse_text">
         <property name="text">
          <string>Impulse:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Impulse_value">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>381</width>
      <height>187</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="geometryLabel">
         <property name="text">
          <string>Geometry:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="geometryComboBox">
         <item>
          <property name="text">
           <string>Hourglass</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Cells:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_Nx">
         <item row="0" column="1">
          <widget class="QSpinBox" name="cells_Nx_SpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="cells_Nx_Label">
           <property name="text">
            <string>N_x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_Ny">
         <item row="0" column="0">
          <widget class="QLabel" name="cells_Ny_Label">
           <property name="text">
            <string>N_y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="cells_Ny_SpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_Nz">
         <item row="0" column="0">
          <widget class="QLabel" name="cells_Nz_Label">
           <property name="text">
            <string>N_z</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="cells_Nz_SpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Particle_number_title">
           <property name="text">
            <string>Particle number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Particle_diameter_title">
           <property name="text">
            <string>Particle diameter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Drag_coefficient_title">
           <property name="text">
            <string>Drag coeff:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="Particle_number_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Particle_diameter_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Drag_coefficient_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QSlider" name="Particle_number_slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>30000</number>
           </property>
           <property name="pageStep">
            <number>1000</number>
           </property>
           <property name="sliderPosition">
            <number>2000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Particle_diameter_slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Drag_coefficient_slider">
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="CustomOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>711</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation_Window">
    <property name="title">
     <string>Simulation Window</string>
    </property>
   </widget>
   <addaction name="menuSimulation_Window"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>CustomOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>geometryComboBox</tabstop>
  <tabstop>cells_Nx_SpinBox</tabstop>
  <tabstop>cells_Ny_SpinBox</tabstop>
  <tabstop>cells_Nz_SpinBox</tabstop>
  <tabstop>Particle_number_slider</tabstop>
  <tabstop>Particle_diameter_slider</tabstop>
  <tabstop>Drag_coefficient_slider</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
